### PR TITLE
Added more Mission Titles

### PR DESCRIPTION
**Pull request by DrDone1**
Added more Tiles for the missions, hope you like them :)

### CODE REVIEW

File: common/src/main/resources/example_config.json [hunk 1]
```diff
@@ -91,7 +104,10 @@
       "Minecrafty mayhem",
       "Terrain takedown",
       "Debris dozer",
-      "A good project's starters"
+      "A good project's starters",
+      "Smash and Grab",
+      "Hammer Time",
+      "Minecraft Mayhem"
```
Comment (muriplz) [inline common/src/main/resources/example_config.json:110]: there is an almost equal one 6 rows above

File: common/src/main/resources/example_config.json [hunk 2]
```diff
@@ -45,7 +45,12 @@
       "Pochita",
       "Chainsaw man",
       "Risky endeavour",
-      "Safety first"
+      "Safety first",
+      "Precision Incision",
```
Comment (muriplz) [inline common/src/main/resources/example_config.json:49]: I prefer the titles to only have the first capital letter, like "Precision incision"

**Review by muriplz — COMMENTED** [review]
There are a few titles that are way too generic, but most of them are good!

File: common/src/main/resources/example_config.json (hunk 2 above)
Comment (DrDone1) [reply common/src/main/resources/example_config.json:49]: fair enough

**Pull request merged**